nomad/event_sink_manager.go

```diff
@@ -217,15 +222,15 @@ func (m *SinkManager) updateSinkProgress() error {
 	return nil
 }
 
+// removeSink removes a sink from the manager's subscriptions.
+// removeSink should only be called from Run
 func (m *SinkManager) removeSink(id string) {
```

Comment:
Given this is now a one line method, with only one caller, and it has a particular requirement for when it can be safely called, it might be better just to inline this method.